package.json

```diff
@@ -46,11 +54,13 @@
     "jsonwebtoken": "^8.5.1",
     "localstorage-polyfill": "^1.0.1",
     "lodash": "^4.17.20",
-    "massive": "^6.6.2",
+    "massive": "^6.6.4",
```

Comment:
I tried both 6.6.4 and the original 6.6.2 and neither would validate the password for the db, don't know what's going on